google-cloud-bigquerystorage/clirr-ignored-differences.xml

```diff
@@ -12,4 +12,9 @@
         <className>com/google/cloud/bigquery/storage/v1alpha2/StreamWriter</className>
         <method>boolean awaitTermination(long, java.util.concurrent.TimeUnit) </method>
     </difference>
-</differences>
\ No newline at end of file
+    <difference>
+        <differenceType>7004</differenceType>
+        <className>com/google/cloud/bigquery/storage/v1alpha2/SchemaCompact</className>
```

Comment:
Instead, you can just add a method that takes in the extra parameter and keep the allowUnknownFields to be by default false on the original function.